timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -13,6 +13,9 @@
 // The number of threads per block for the setting of the final weight of the moved mol is low
 // if using the same number as in the rest of the kernels of DEFAULT_THREADS_PER_BLOCK
 #define WEIGHT_THREADS_PER_BLOCK 512
+// The number of translations to generate each step. The first three values are a unit vector translation and the fourth
+// value is used for the metropolis hasting check
+#define TRANSLATIONS_PER_STEP 4
```

Comment:
okay so for BD EXCHANGE, `TRANSLATIONS_PER_STEP = 4` and is `#define`, defined by`xyzw`

Where as for TIBD EXCHANGE, `TRANSLATIONS_PER_STEP = 6`, defined by `x_inner, y_inner, z_inner, x_outer, y_outer, z_outer`

Can we *please*:

1) have them *both* be #defines or both be `const static int`s
2) name them separately, i.e. `BD_TRANSLATIONS_PER_STEP_XYZW` and `TIBD_TRANSLATIONS_PER_STEP_XYZXYZ` to be verbose

Reply:
I know it's technically safe to have them have the same name but in different `.cu` files, but it's easier for readers to grep when the names are different, and to avoid hell breaking lose when one of these things get moved into a header by a future developer

Reply:
Now consistently using `const static int` in https://github.com/proteneer/timemachine/pull/1234/commits/7bf3826bb6fb5f42c8021fe7b118415a3b3f0f03
Renamed to be more explicit in https://github.com/proteneer/timemachine/pull/1234/commits/441fcde9b98a2b65f6cf81ba0724fd2968c329fa